### DISCUSSION TITLE
CSS variables generated even not used

### DISCUSSION THREAD

**Discussion by Yovach** [chakra-ui/panda#1887] [General]
### Description

When I use Panda CSS (0.23.0) with Next.js (14.0.4), all the CSS variables are generated, even those that are not used.

### Link to Reproduction

https://play.panda-css.com/8RAroJwVFG

### Steps to reproduce

1. Go to the CSS bundle
2. All the CSS variables are generated

### JS Framework

Next.js

### Panda CSS Version

0.23.0

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by astahmer** [comment]
this is intended, as those variables could be used in your JS directly for example [with the `token` function](https://panda-css.com/docs/guides/dynamic-styling#using-tokenvar)

that being said, you can do it anyway with a postcss plugin like https://www.npmjs.com/package/postcss-prune-var

**Reply by Yovach** [reply]
Oh OK, thanks!
